guides/v2.2/cloud/release-notes/cloud-tools.md

```diff
@@ -22,7 +22,41 @@ php ./vendor/bin/ece-tools list
 ```
 
 The following updates describe the latest improvements to the `{{site.data.var.ct}}` package, which uses the following version sequence: `200<major>.<minor>.<patch>`. See [Upgrades and patches]({{ site.baseurl }}/guides/v2.1/cloud/project/project-upgrade-parent.html) for information about updating to the latest release of the `{{site.data.var.ct}}` package.
-	
+
+## v2002.0.15
+
+#### New features
+
+-  **Docker Updates**
+
+    -  <!-- MAGECLOUD-2799 -->Added PHP 7.2 image—Added support for PHP 7.2 in Cloud Docker; updated the [Launch Docker configuration]({{ page.baseurl }}/cloud/docker/docker-config.html) to include the `docker:build --php` option to specify the version of PHP compatible with your Magento Commerce version.
+
+    -  <!-- MAGECLOUD-2565 -->Added a [Cron container]({{page.baseurl}}/cloud/docker/docker-development.html#cron-container) based on the PHP-CLI image.
+
+    -  Added the following services to the Docker build:
+
+        -  <!-- MAGECLOUD-2567 & 2889-->RabbitMQ 3.5 and 3.7
+        -  <!-- MAGECLOUD-2569 & 2887 -->ElasticSearch 1.7 and 2.4
+        -  <!-- MAGECLOUD-2886 -->Redis 3.2 and 4.0
+
+    -  <!-- MAGECLOUD-2577 -->Now you have the DB dump capability when using Cloud Docker [database container]({{page.baseurl}}/cloud/docker/docker-development.html#database-container). Also, you can [share files]({{page.baseurl}}/cloud/docker/docker-development.html#sharing-data-between-host-machine-and-container) between a host machine and a container using the `docker/mnt` directory.
+
+-  <!-- MAGECLOUD- 2575 -->**Configure with PHP constants**—Added support for [PHP constants]({{page.baseurl}}/cloud/project/magento-env-yaml.html#php-constants) in the `.magento.env.yaml` configuration file.
+
+-  <!--MAGECLOUD-2879-->**New Environment variable**—By default, only the Production environment has Google Analytics enabled. You can enable Google Analytics on the Staging and Integration environments using the  [ENABLE_GOOGLE_ANALYTICS]({{page.baseurl}}/cloud/env/variables-deploy.html#enable_google_analytics) environment variable.
+
+#### Resolved Issues
+
+-  <!-- MAGECLOUD-2862 -->Fixed an issue involving cron processes that prevented the start of the post-deploy phase, when enabled. Now, if you have the post-deploy hook enabled, the cron processes are enabled again at the beginning of the post-deploy phase.
```

Comment:
not sure what "when enabled" refers to .... please check

Reply:
You make a good point. I think I have been too broad with that, and not just here. That phase is not available to every project. Rethinking...